desktop/components/article/test/view.coffee

```diff
@@ -210,23 +151,11 @@ describe 'ArticleView', ->
 
     it 'calls fillwidth on images', ->
       @view.refreshWindowSize()
-      @fillwidth.callCount.should.be.above 1
+      @fillwidth.callCount.should.equal 1
 
     it 'calls setupMaxImageHeights on single images', ->
       @view.refreshWindowSize()
-      @setupMaxImageHeights.callCount.should.be.above 1
-
-  describe '#embedMobileHeight', ->
-
-    it 'sets iframe height to desktop height on large screens', ->
-      @view.windowWidth = 1250
-      @view.embedMobileHeight()
-      @view.$el.find('iframe').height().should.equal 600
-
-    it 'sets iframe height to mobile height on small screens', ->
-      @view.windowWidth = 400
-      @view.embedMobileHeight()
-      @view.$el.find('iframe').height().should.equal 1100
+      @setupMaxImageHeights.callCount.should.equal 1
 
   xdescribe '#resetImageSetPreview', ->
 
```

Comment:
What's up with this one being x'd?

Reply:
Not sure, it was X-ed a while ago. I just turned it back on tho & it ended up being a simple fix.